### PR TITLE
Namespace V1 hooks and constants

### PR DESCRIPTION
**Pull request by aeolianeth**
## What does this PR do and why?

_another one_:

- Move any v1-specific hooks to `hooks/v1/`.
- Moves any v1-specific constants files to `constants/v1/`.
- Defines some missing project IDs and uses them where necessary.
- Removes `src/hooks/EventListener.ts` file (it's unused)

## Screenshots or screen recordings

N/A

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](../CONTRIBUTING.md#approval-guidelines) for this PR.
- [-] I have tested this PR in [all supported browsers](../CONTRIBUTING.md#browser-support).

### CODE REVIEW

**Comment by aeolianeth** [comment]
@peripheralist for your review!

I haven't renamed the hooks themselves (i.e. `useV1...`). It didn't feel like it made sense in some cases , but I don't really have a strong opinion. Happy to make that change if we think it's more clear. cc @johnnyd-eth for additional thoughts

**Comment by peripheralist** [comment]
love this! gonna dig through it before approving, but seems conceptually sound 👌. agree that putting `v1` in the hook names might not be necessary

File: src/constants/v1/archivedProjects.ts [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
+  SVSPOOL002,
+  FIDENZA420,
+  SANTA_DAO,
+  INFLATIONLESS_DAO,
+  VOTING_DAO,
+} = V1_PROJECT_IDS
 
 const archivedProjectIdsByNetwork: Partial<Record<NetworkName, number[]>> = {
   [NetworkName.mainnet]: [
-    11, // @pxdao
```
Comment (peripheralist) [inline src/constants/v1/archivedProjects.ts:7]: yesssss 👏

**Review by peripheralist — APPROVED** [review]
codebase feeling so clean!

this is awesome. great step towards v2 support 👏

**Pull request merged**